README.md

```diff
@@ -250,9 +250,9 @@ unmock. nock('http://myapp.iriscouch.com')
   })
 ```
 
-### Passthrough
+### Trivial API calls
```

Comment:
I also wonder if there's a better header for this... we can keep for the sake of this PR, but might revisit later.

Reply:
I just changed it to "Ignorable", which is perhaps not as easy for a non-native speaker as it is a neologism, but it may get the idea across better.

Reply:
Yeah maybe... let's sit on it for a while and if something else comes to mind we can always change it.

Merging now 🚀